### PR TITLE
Client question updated

### PR DESCRIPTION
**Pull request by gouravmalviya061290**

### CODE REVIEW

File: yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java [hunk 1]
```diff
@@ -33,28 +34,32 @@ public class ClientQuestionBankController {
...
 		}
-		ClientQuestion savedQuestion = clientQuestionBankService.saveOrUpdate(question);
-		return new ResponseEntity<ClientQuestion>(savedQuestion, HttpStatus.CREATED);
+		List<ClientQuestion> savedQuestion = clientQuestionBankService.saveall(list);
+		return new ResponseEntity<>(savedQuestion, HttpStatus.CREATED);
 	}
 
-	@GetMapping("/questions")
+	@GetMapping("/clientQuestions")
 	public Iterable<ClientQuestion> getAllQuestions() {
```
Comment (ramverma-github) [inline yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java:51]: Please use getClientQuestions as method name

File: yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java [hunk 2]
```diff
@@ -33,28 +34,32 @@ public class ClientQuestionBankController {
...
 		return clientQuestionBankService.findAllQuestion();
 	}
-	
+
 	@PostMapping("/clientQuestionUpload")
 	public ResponseEntity<?> uploadClientExcelFile(@RequestParam("file") MultipartFile file) {
```
Comment (ramverma-github) [inline yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java:56]: please use uploadClientQuestionExcelFile as mthod name

**Review by ramverma-github — COMMENTED** [review]
Please resolve commets

File: yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java (hunk 2 above)
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java:56]: ok

File: yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java (hunk 1 above)
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java:51]: ok

**Review by ramverma-github — APPROVED** [review]
Looks good.

**Pull request merged**